src/components/Match/MatchStory.jsx

```diff
@@ -13,6 +13,16 @@ import styles from './Match.css';
 
 const heroesArr = jsonFn(heroes);
 
+const itemQualityColor = {
+  rare: '#1a87f9',
+  artifact: '#e29b01',
+  secret_shop: '#ffffff',
+  consumable: '#1d80e7',
+  common: '#2bab01',
+  epic: '#b812f9',
+  component: '#ffffff',
+};
```

Comment:
If you think it might be useful to other developers/that this might be reused somewhere, you can add it as a json file in https://github.com/odota/dotaconstants

If you do that, then I'll publish a new version of dotaconstants and you can do `npm update` to update it locally, then reference this blob like we do heroes, items, etc.